sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -388,6 +388,8 @@ class SessionCatalog(
   private def makeQualifiedTablePath(locationUri: URI, database: String): URI = {
     if (locationUri.isAbsolute) {
       locationUri
+    } else if (locationUri.getPath.startsWith("/")) {
```

Comment:
Use `new Path(locationUri).isAbsolute`?

Reply:
Thanks! Will change to follow your suggestion.